### ISSUE TITLE
Warehouse door blocked by street light

### ISSUE THREAD

**Issue by sparr** [CleverRaven/Cataclysm-DDA#61292]
### Describe the bug

The roll up vehicle door on a warehouse can have a street light placed in front of it. This makes driving in and out difficult, and feels unrealistic.

### Steps to reproduce

1. Find a warehouse adjacent to a street with street lights

### Expected behavior

Something like one of these
* no street light in front of the warehouse
* lane marking on the sidewalk
* door split into two doors like an auto shop

### Screenshots

![Screenshot_2022-09-26_11-30-28](https://user-images.githubusercontent.com/121299/192323981-c29a078f-6198-43ef-b7a0-bb1059073fdd.png)

### Versions and configuration

- OS: Linux
    - OS Version: LSB Version: n/a; Distributor ID: ManjaroLinux; Description: Manjaro Linux; Release: 22.0.0; Codename: Sikaris; 
- Game Version: 0.C-73702-ge615a9db84 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

**Comment by Night-Pryanik** [comment]
Duplicate of #50279.
